waddrmgr/manager_test.go

```diff
@@ -1264,13 +1264,6 @@ func testRenameAccount(tc *testContext) bool {
 	if !checkManagerError(tc.t, testName, err, wantErrCode) {
 		return false
 	}
-	// Test account name validation
-	testName = "*"
```

Comment:
or not.  using "imported" always will fail with ErrDupliateAccount so it will never check for a reserved name.  Seems backwards but oh well.